test/TestUtils.py

```diff
@@ -526,7 +526,34 @@ def test_nested_items() -> None:
         ("list-item", "apple", "fruits"),
         ("list-item", "orange", "fruits"),
     ]
-    assert list(utils.nested_items(data)) == items
+    with pytest.deprecated_call():
```

Comment:
It's always recommended to make use of the `match=` argument in such cases so that the test is accurate.

Reply:
done.